src/resources/api/announcement/comment/CreateComment.js

```diff
@@ -0,0 +1,23 @@
+import api from "../../config";
+
+let apiPrefix = "/badaso-api";
```

Comment:
Suggestion: move this variable to a file (e.g. `constants.js`) in the next sprint, since it's redeclared everywhere in the codebase